--- a/config/cmake/cacheinit.cmake	2021-04-12 17:39:46.000000000 -0500
+++ b/config/cmake/cacheinit.cmake	2021-03-01 13:20:47.000000000 -0600
@@ -65,7 +65,7 @@
 # bitshuffle
 ###########
 
-set (BSHUF_GIT_URL "https://git@bitbucket.hdfgroup.org/scm/test/bitshuffle.git" CACHE STRING "Use BSHUF from HDF repository" FORCE)
+set (BSHUF_GIT_URL "https://github.com/hyoklee/bitshuffle.git" CACHE STRING "Use BSHUF from HDF repository" FORCE)
 set (BSHUF_GIT_BRANCH "master" CACHE STRING "" FORCE)
 
 set (BSHUF_TGZ_NAME "bitshuffle.tar.gz" CACHE STRING "Use BSHUF from compressed file" FORCE)
@@ -83,19 +83,29 @@
 
 set (BLOSC_PACKAGE_NAME "blosc" CACHE STRING "Name of BLOSC package" FORCE)
 
-set (ZLIB_GIT_URL "https://git@bitbucket.hdfgroup.org/scm/test/zlib.git" CACHE STRING "Use ZLIB from HDF repo" FORCE)
+#######
+# zlib
+#######
+set (ZLIB_GIT_URL "https://github.com/hyoklee/zlib.git" CACHE STRING "Use ZLIB from HDF repo" FORCE)
 set (ZLIB_GIT_BRANCH "master" CACHE STRING "" FORCE)
-
 set (ZLIB_TGZ_NAME "ZLib.tar.gz" CACHE STRING "Use ZLib from compressed file" FORCE)
-
 set (ZLIB_PACKAGE_NAME "zlib" CACHE STRING "Name of ZLIB package" FORCE)
 
 #######
+# szip
+#######
+# HDF5 1.13 uses libaec instead of szip.
+set (SZIP_GIT_URL "https://github.com/hyoklee/libaec.git" CACHE STRING "Use SZIP from HDF repo" FORCE)
+set (SZIP_GIT_BRANCH "main" CACHE STRING "" FORCE)
+
+
+#######
 # bzip2
 ######
 #
-set (BZ2_GIT_URL "https://git@bitbucket.hdfgroup.org/scm/test/bzip2.git" CACHE STRING "Use BZ2 from HDF repository" FORCE)
-set (BZ2_GIT_BRANCH "master" CACHE STRING "" FORCE)
+set (BZ2_GIT_URL "https://github.com/hyoklee/bzip2.git" CACHE STRING "Use BZ2 from HDF repository" FORCE)
+# set (BZ2_GIT_BRANCH "master" CACHE STRING "" FORCE)
+set (BZ2_GIT_BRANCH "main" CACHE STRING "" FORCE)
 
 set (BZ2_TGZ_NAME "BZ2.tar.gz" CACHE STRING "Use BZ2 from compressed file" FORCE)
 
@@ -116,8 +126,9 @@
 # jpeg
 ######
 
-set (JPEG_GIT_URL "https://git@bitbucket.hdfgroup.org/scm/test/jpeg.git" CACHE STRING "Use JPEG from HDF repository" FORCE)
-set (JPEG_GIT_BRANCH "jpeg9c" CACHE STRING "" FORCE)
+set (JPEG_GIT_URL "https://github.com/hyoklee/jpeg.git" CACHE STRING "Use JPEG from HDF repository" FORCE)
+# set (JPEG_GIT_BRANCH "jpeg9c" CACHE STRING "" FORCE)
+set (JPEG_GIT_BRANCH "main" CACHE STRING "" FORCE)
 
 #set (JPEG_TGZ_NAME "JPEG9c.tar.gz" CACHE STRING "Use JPEG from compressed file" FORCE)
 set (JPEG_TGZ_NAME "JPEG.tar.gz" CACHE STRING "Use JPEG from compressed file" FORCE)
@@ -130,7 +141,7 @@
 
 set (BUILD_LZ4_LIBRARY_SOURCE ON CACHE BOOL "build the lz4 library within the plugin" FORCE)
 
-set (LZ4_GIT_URL "https://git@bitbucket.hdfgroup.org/scm/test/lz4.git" CACHE STRING "Use LZ4 from HDF repository" FORCE)
+set (LZ4_GIT_URL "https://github.com/hyoklee/lz4.git" CACHE STRING "Use LZ4 from HDF repository" FORCE)
 set (LZ4_GIT_BRANCH "master" CACHE STRING "" FORCE)
 
 set (LZ4_TGZ_NAME "lz4.tar.gz" CACHE STRING "Use LZ4 from compressed file" FORCE)
@@ -141,7 +152,7 @@
 # lzf
 ######
 
-set (LZF_GIT_URL "https://git@bitbucket.hdfgroup.org/scm/test/lzf.git" CACHE STRING "Use LZF from HDF repository" FORCE)
+set (LZF_GIT_URL "https://github.com/hyoklee/lzf.git" CACHE STRING "Use LZF from HDF repository" FORCE)
 set (LZF_GIT_BRANCH "master" CACHE STRING "" FORCE)
 
 set (LZF_TGZ_NAME "lzf.tar.gz" CACHE STRING "Use LZF from compressed file" FORCE)
@@ -152,9 +163,9 @@
 # mafisc
 ########
 
-#set (BUILD_MAFISC_LIBRARY_SOURCE OFF CACHE BOOL "build the mafisc library within the plugin" FORCE)
+# set (BUILD_MAFISC_LIBRARY_SOURCE OFF CACHE BOOL "build the mafisc library within the plugin" FORCE)
 
-#set (MAFISC_PACKAGE_NAME "mafisc" CACHE STRING "Name of MAFISC package" FORCE)
+set (MAFISC_PACKAGE_NAME "mafisc" CACHE STRING "Name of MAFISC package" FORCE)
 
 ######
 # szf
@@ -178,3 +189,37 @@
 
 set (ZFP_PACKAGE_NAME "zfp" CACHE STRING "Name of ZFP package" FORCE)
 
+######
+# zstd
+######
+set (ZSTD_GIT_URL "https://github.com/hyoklee/zstd.git" CACHE STRING "Use ZSTD from Github" FORCE)
+set (ZSTD_GIT_BRANCH "master" CACHE STRING "" FORCE)
+
+set (ZSTD_TGZ_NAME "zstd.tar.gz" CACHE STRING "Use ZSTD from compressed file" FORCE)
+
+set (ZSTD_PACKAGE_NAME "zstd" CACHE STRING "Name of ZSTD package" FORCE)
+
+##########
+# bitgroom
+##########
+# set (BITGROOM_GIT_URL "https://github.com/hyoklee/bitgroom.git" CACHE STRING "Use BITGROOM from Github" FORCE)
+# set (BITGROOM_GIT_BRANCH "master" CACHE STRING "" FORCE)
+
+# set (BITGROOM_TGZ_NAME "bitgroom.tar.gz" CACHE STRING "Use BITGROOM from compressed file" FORCE)
+
+set (BITGROOM_PACKAGE_NAME "bitgroom" CACHE STRING "Name of BITGROOM package" FORCE)
+
+###########################
+# Pass-through external VOL
+###########################
+set (PV_PACKAGE_NAME "pv" CACHE STRING "Name of PV package" FORCE)
+
+###########
+# Async VOL
+###########
+set (AV_PACKAGE_NAME "av" CACHE STRING "Name of AV package" FORCE)
+
+###########
+# Cache VOL
+###########
+set (CV_PACKAGE_NAME "cv" CACHE STRING "Name of CV package" FORCE)
